advanced_source/torch_script_custom_ops.rst

```diff
@@ -1076,3 +1076,34 @@ visible to TorchScript:
   >>> torch.ops.load_library("warp_perspective.so")
   >>> print(torch.ops.custom.warp_perspective)
   <built-in method custom::warp_perspective of PyCapsule object at 0x7ff51c5b7bd0>
+
+
+FAQ
+---------------------------------------
```

Comment:
cc: @suo 

@suo I'm going to merge this in, please make sure you update this when we add support for any more types.